docs/editor/vscode-web.md

```diff
@@ -175,7 +175,9 @@ Edge and Chrome today support the [File System API](https://developer.mozilla.or
 
 ### Browser support
 
-You can use VS Code for the Web in Chrome, Edge, Firefox, and Safari.
+You can use VS Code for the Web in the latest versions of Chrome, Edge, Firefox, and Safari. Older versions of each browser may not work - we only guarantee support for the latest version. 
+
+> **Tip:** A good indication of which browser version will work is the version of [Playwright](https://playwright.dev/) we're using in VS Code. The browser versions it comes with is what our tests assume exist. You can check out the VS Code repo's [`package.json`](https://github.com/microsoft/vscode/blob/main/package.json) to find the version of Playwright in `devDependencies/@playwright/test`.
```

Comment:
How about:
"One way to check the compatible browser version is to look at the version of [Playright]((https://playwright.dev) currently used for testing VS Code and review its supported browser versions. You can find the currently used Playwright version in the VS Code repo's [package.json](https://github.com/microsoft/vscode/blob/main/package.json) file at `devDependencies/@playwright/test`. Once you know the Playwright version, for example `1.37`, you can then review the **Browser Versions** section in their [Release notes](https://playwright.dev/docs/release-notes)."